common/lib/dependabot/pull_request_creator.rb

```diff
@@ -214,12 +214,6 @@ def labeler
         )
     end
 
-    def library?
```

Comment:
Not sure this is needed here as we only seem to care if the requirement has changed or we have a previous version.